### ISSUE TITLE
Addon falsely alerts in PHP about unclosed parentheses

### ISSUE THREAD

**Issue by Pawwwle** [niccord/BracketsCheck#15]
Hallo! I am using N++ latest 64 bits version. The plugin BracketsCheck does not work. I tested version 1.2.2 and 1.2.3. The addon falsely alerts in PHP about unclosed parentheses.

**Comment by niccord** [comment]
Hi @Pawwwle,
I originally developed this plugin to check parenthesis in large SQL files, so it is not well-tested with PHP files.

Please provide a minimal reproducible example of the text you are using and the error the plugin returns you.

Cheers.
Nicola

**Comment by Pawwwle** [comment]
```
function phpbb_optionset($bit, $set, $data)
{
	if ($set && !($data & 1 << $bit))
	{
		$data += 1 << $bit;
	}
	else if (!$set && ($data & 1 << $bit))
	{
		$data -= 1 << $bit;
	}

	return $data;
}
```
![Zrzut ekranu 2023-02-11 115117](https://user-images.githubusercontent.com/115624603/218254372-81fb5a65-bc2b-4bbc-88e0-232e9e6e16f9.png)

**Comment by niccord** [comment]
Have you tried disabling the angle brackets check in the plugin menu?

**Comment by Pawwwle** [comment]
Yes. An error occurs when checking angle brackets. Can you fix it?

**Comment by niccord** [comment]
I haven't had time to check it yet, what's the error?
